lib/new_relic/agent/instrumentation/active_record_helper.rb

```diff
@@ -93,9 +93,21 @@ def pluck(*args, &blk)
 
         ACTIVE_RECORD = "ActiveRecord".freeze
         OTHER = "other".freeze
+        MAKARA_SUFFIX = "_makara".freeze
+
+        # convert vendor (makara, etc.) wrapper names to their bare names
+        # ex: postgresql_makara -> postgresql
+        def bare_adapter_name(adapter_name)
```

Comment:
Route all adapter name look ups through this choke point. For now it only has support for Makara, to strip away the `_makara` suffix on an adapter name, but in future we could leverage this method to support other gems / adapters as well.